docs/FAQ/index.md

```diff
@@ -28,135 +36,138 @@ directory with product configuration and user groups in the Admin Console
+a problem?
+
+For example,  the back-off can stop the activity to query or update 
 in X number of tries. This can leave a partially updated list of users or a 
-partial query.<br/>
-A: No, User sync handles throttling and retries so that throttling may slow 
+partial query.
+
+No, User sync handles throttling and retries so that throttling may slow 
 down the overall user sync process, but there is no problem caused by throttling 
 and user sync will properly complete all operations.
  
-**Q:  Is there a local list of users created/updated (on the user sync side) 
+###  Is there a local list of users created/updated (on the user sync side) 
 in order not to disturb our server every time for a list of users from the 
-org/product configuration?**<br/>
-A: No, User sync always queries the Adobe user management systems to get 
+org/product configuration?
+
+No, User sync always queries the Adobe user management systems to get 
 current information when it is run.
 
-**Q: I’d like to know more on how the user sync tool deals with the throttling 
-in general.**<br/>
-A: The Adobe systems protect themselves from overload by tracking the incoming 
+### I’d like to know more on how the user sync tool deals with the throttling 
+in general.
+
+The Adobe systems protect themselves from overload by tracking the incoming 
 request volume.  If this is starting to exceed limits, then requests return 
-a "retry-after" header indicating when capacity will be available again.  User sync honors these headers and waits for the requested amount of time before retrying.  More information, including code samples, can be found in the User Management API documentation at https://www.adobe.io/apis/cloudplatform/usermanagement/docs/throttling.html.
+a "retry-after" header indicating when capacity will be available again.  User sync honors these headers and waits for the requested amount of time before retrying.  More information, including code samples, can be found in the [User Management API documentation](https://www.adobe.io/apis/cloudplatform/usermanagement/docs/throttling.html).
  
-**Q: Is the user sync tool dedicated for federated Ids or any type of ID can be 
-created?**<br/>
-A: User sync supports all id types (Adobe IDs, Federated IDs and Enterprise IDs).
+### Is the user sync tool limited to federated Ids or can any type of ID can be 
+created?
 
-**Q: An org can have owned domains and domains to which another org has granted it 
+User sync supports all id types (Adobe IDs, Federated IDs and Enterprise IDs).
+
+### An org can have owned domains and domains to which another org has granted it 
 access. The UM API can run queries on both. How about user sync? Users are 
 added to the product configuration just because they are found in the AD group, 
-no check on the domain?**<br/>
-A: User Sync uses the UMAPI, so it can both query and manage the user 
+no check on the domain?
+
+User Sync uses the UMAPI, so it can both query and manage the user 
```

Comment:
this answer is incorrect.  Both User Sync and the Admin Console have exactly the same restrictions with respect to users in non-owned domains: they can be created but not updated.  I believe you should mention that User Sync is designed to be used so that it is given credentials for the owning org of the domain as primary, and then can be given credentials for other orgs where groups should be managed, but that it assumes those other orgs are trusting the users, not owning them.